mmengine/structures/base_data_element.py

```diff
@@ -561,6 +562,16 @@ def to_dict(self) -> dict:
             for k, v in self.all_items()
         }
 
+    def pin_memory(self) -> 'BaseDataElement':
```

Comment:
Need to provide `is_pinned` method

Reply:
There are many properties inside this object, so if one is not pinned, return false?